pages/jdl.md

```diff
@@ -1,1447 +0,0 @@
-  - `with jpaDerivedIdentifier` whether `@MapsId` is used for the association (applicable only for one-to-one)
-
-Here's a simple example:
-
-    entity Book
-    entity Author
-
-    relationship ManyToOne {
-      Book to Author
-    }
-
-Declaring an injected field is optional, as one (or both) is set by default as needed.
-The previous example is equivalent to this one:
-
-    entity Book
-    entity Author
-
-    relationship ManyToOne {
-      Book{author} to Author
-    }
-
-Let's make it more complex.
-A Book has one required Author, an Author has several Books.
-
-    entity Book
-    entity Author {
-      name String required
-    }
-
-    relationship OneToMany {
-      Author{book} to Book{writer(name) required}
-    }
-
-Here, the `Book` class will have a **required** field named `writer` that will be linked through the `name` field of `Author`.
-
-Of course, in real cases, you'd have a lot of relationships and always writing the same three lines could be tedious.
-That's why you can declare something like:
-
-```
-entity A
-entity B
-entity C
-entity D
-
-relationship OneToOne {
-  A{b} to B{a}
-  B{c} to C
-}
-relationship ManyToMany {
-  A{d} to D{a}
-  C{d} to D{c}
-}
-```
-
-The join is always done using the `id` field which is also the default field shown when editing a relation in the
-front-end. If another field should be shown instead, you can specify it like this:
-
-```
-entity A {
-  name String required
-}
-entity B
-
-
-relationship OneToOne {
-  A{b} to B{a(name)}
-}
-```
-
-JPA Derived Identifier - @MapsId can be declared as following which is currently only supported for one-to-one:
-
-```
-entity A {
-  name String required
-}
-entity B
-
-
-relationship OneToOne {
-  A{b} to B{a(name)} with jpaDerivedIdentifier
-}
-```
-
-
-This makes JHipster generate a REST resource that returns both the `id` and `name` of the linked entity to the front-end,
-so the name can be shown to the user instead.
-
-### <a name="enumerationdeclaration"></a> Enumerations
-
-To make Enums with JDL just do as follows:
-
-- Declare an Enum where you want in the file:
-
-        enum Language {
-          FRENCH, ENGLISH, SPANISH
-        }
-
-- In an entity, add fields with the Enum as a type:
-
-        entity Book {
-          title String required,
-          description String,
-          language Language
-        }
-
-
-### <a name="blobdeclaration"></a> Blob (byte[])
-JHipster gives a great choice as one can choose between an image type or any binary type. JDL lets you do the same.
-Just create a custom type (see DataType) with the editor, name it according to these conventions:
-  - `AnyBlob` or just `Blob` to create a field of the "any" binary type;
-  - `ImageBlob` to create a field meant to be an image.
-  - `TextBlob` to create a field for a CLOB (long text).
-
-And you can create as many DataTypes as you like.
-
-
-### <a name="entityoptiondeclaration"></a> Entity option declaration
```

Comment:
I think the explanation of each options should be retained under the options section

Reply:
They are, aren't they? I think I misunderstand what you mean.

Reply:
Here each option is explained, but in new options page there is just an example of usage but no explanation

Reply:
Ah okay! I'll update it